policyengine_us/parameters/gov/states/ar/tax/income/deductions/standard.yaml

```
@@ -0,0 +1,23 @@
+description: Arkansas provides a standard deduction of this amount.
+metadata:
+  label: Arkansas standard deduction
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Code of Arkansas Public Access A.C.A. Subchapter 4. § 26-51-430  
+    href: https://advance.lexis.com/container/?pdmfid=1000516&crid=1b7c59fc-ee50-4b71-9f32-869742d3b94c&func=LN.Advance.ContentView.getFullToc&nodeid=ABAAAFAAC&typeofentry=Breadcrumb&config=00JAA3ZTU0NTIzYy0zZDEyLTRhYmQtYmRmMS1iMWIxNDgxYWMxZTQKAFBvZENhdGFsb2cubRW4ifTiwi5vLw6cI1uX&action=publictoc&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A649P-FBD0-R03K-54BT-00008-00&pdtocfullpath=%2Fshared%2Ftableofcontents%2Furn%3AcontentItem%3A50XD-80G1-DY3X-63F1-00008-00&ecomp=-3vckkk&prid=e3cf7c68-667d-45c5-b52a-91f7e551c867
+  - title: Arkansas 2022 Individual Income Tax Forms and Instructions
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=14
+
+JOINT:
+  2022-01-01: 4_540
```

Since you cite the law
```suggestion
  # $4,400 in 2015 + COLA, rounded by $10.
  2022-01-01: 4_540
```